include/States.h

```diff
@@ -104,7 +106,7 @@ class States {
    */
     std::vector<int> hermiteIndices(double tolerance, std::vector <int> indexList,
                                     double baseTime, double timeScale);
-    std::vector<ale::State> m_states; //! Represent at states internally to keep pos, vel together
+    std::vector<State> m_states; //! Represent at states internally to keep pos, vel together
```

Comment:
at -> as. I know this wasn't your typo. It was probably mine, but while you're in here, might as well fix it.